ix-dev/community/nzbget/templates/test_values/basic-values.yaml

```
@@ -0,0 +1,31 @@
+resources:
+  limits:
+    cpus: 2.0
+    memory: 4096
+
+nzbget:
+  additional_envs: []
+network:
+  host_network: false
+  web_port: 6789
```

```suggestion
  web_port: 8080
```
So you can test that it works with non-default port.